stellarphot/io/tess.py

```diff
@@ -18,13 +18,61 @@
 __all__ = ["TessSubmission", "TOI", "TessTargetFile"]
 
 # Makes me want to vomit, but....
-DEFAULT_TABLE_LOCATION = "who.the.fuck.knows"
+DEFAULT_TABLE_LOCATION = "who.the.heck.knows"
 TOI_TABLE_URL = "https://exofop.ipac.caltech.edu/tess/download_toi.php?output=csv"
 TIC_regex = re.compile(r"[tT][iI][cC][^\d]?(?P<star>\d+)(?P<planet>\.\d\d)?")
 
 
 @dataclass
 class TessSubmission:
+    """ A data class to represent TESS submissions.
+
+    Parameters
+    ----------
+
+    telescope_code: String
+        The telescope code, e.g. "SRO" or "TJO"
+
+    filter: String
+        The filter used for the observations, e.g. "Ic" or "Rc"
+
+    utc_start: String
+        The UTC date of the first observation, in YYYYMMDD format
+
+    tic_id: int
+        The TIC ID of the target
+
+    planet_number: int
+        The planet number, if applicable
+
+
+    Attributes
+    ----------
+    base_name: String
+        The base name of the submission, e.g. "TIC123456789-01_20200101_SRO_Ic"
+
+    seeing_profile: String
+        The name of the seeing profile file, e.g. "TIC123456789-01_20200101_SRO_Ic_seeing-profile.png"
+
+    field_image: String
+        The name of the field image file, e.g. "TIC123456789-01_20200101_SRO_Ic_field.png"
+
+    field_image_zoom: String
+        The name of the zoomed-in field image file, e.g. "TIC123456789-01_20200101_SRO_Ic_field-zoom.png"
+
+    apertures: String
+        The name of the apertures file, e.g. "TIC123456789-01_20200101_SRO_Ic_measurements.apertures"
+
+    tic_coord: SkyCoord
+        The SkyCoord of the target, from the TIC
```

Comment:
ends abruptly -- from the TIC catalog maybe?

Reply:
Fixed.